public/app/features/trails/DataTrailsApp.tsx

```diff
@@ -114,28 +101,10 @@ export function getDataTrailsApp() {
 function getInitialTrail() {
   const newTrail = newMetricsTrail();
 
-  // Set the initial state of the newTrail based on the URL,
-  // In case we are initializing from an externally created URL or a page reload
-  getUrlSyncManager().initSync(newTrail);
-  // Remove the URL sync for now. It will be restored on the trail if it is activated.
-  getUrlSyncManager().cleanUp(newTrail);
-
   // If one of the recent trails is a match to the newTrail derived from the current URL,
   // let's restore that trail so that a page refresh doesn't create a new trail.
   const recentMatchingTrail = getTrailStore().findMatchingRecentTrail(newTrail)?.resolve();
```

Comment:
We can no longer find `recentMatchingTrail` because we were using `initSync` above to make `newTrail` match the current URL. (due to the removal of 117-122). The trail will remain in it's blank initial state, ignoring any URL state.

The net effect is that we have lost the ability to automatically reload a recent trail after refresh. Instead it will create a new trail with identical state for its second step. As you mentioned in the past, this is arguably a feature (the ability to "blank out" a long trail by reloading.

If we did still want to do a one-time "initialize this trail from the current URL state" maybe there is a more straightforward way other than the removed sequence of `initSync`, `cleanUp`?

Otherwise, we could just drop `recentMatchingTrail` and return `newTrail` from this function.